docs/source/python/development.rst

```diff
@@ -267,6 +267,26 @@ we recommend the following:
 #. Add ``-DARROW_ORC=on`` to the CMake flags.
 #. Add ``--with-orc`` to the ``setup.py`` flags.
 
+
+Building documentation
+----------------------
+
+To build the documentation:
```

Comment:
It may be helpful to explain how to install the required software dependencies, especially Sphinx (`pip install -r requirements.txt` from the `docs` directory).

Reply:
I've added the command

Reply:
I would expect a `pip` command instead, as I suggested above. `pip` works in `conda` environments, but not the reverse, so it's more versatile.

Reply:
Hi It looks pip doesn't hold doxygen, so it might nee to be installed through brew on max or yum/zipper in linux..